.gitignore

```diff
@@ -6,3 +6,4 @@ vendor/
 .DS_Store
 go.sum
 localenvoyconfig.yaml
+localdev/
```

Comment:
Please revert this change. If this is your personal preference then you can add it to a global ~/.gitignore in your workspace.

Reply:
Will do, thanks for the feedback.